setup.py

```diff
@@ -31,6 +31,8 @@
         "pathlib2>=2.2.1,<3",
         "PyYAML",
         "raven",
+        "tqdm",
+        "scandir",
```

Comment:
can you install this only for python less than 3.5? see: https://hynek.me/articles/conditional-python-dependencies/

Reply:
👍